app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :microposts
+  validates :name, presence: true
+  validates :email, presence: true
```

Comment:
This can be simplified

```suggestion
  validates :name, :email, presence: true
```